pkg/logs/config/config.go

```diff
@@ -135,8 +135,10 @@ func buildTCPEndpoints() (*Endpoints, error) {
 }
 
 func buildHTTPEndpoints() (*Endpoints, error) {
+	proxyAddress := coreConfig.Datadog.GetString("logs_config.socks5_proxy_address")
 	main := Endpoint{
-		APIKey: getLogsAPIKey(coreConfig.Datadog),
+		APIKey:       getLogsAPIKey(coreConfig.Datadog),
+		ProxyAddress: proxyAddress,
```

Comment:
Could we rename that field to `Socks5ProxyAddress` or something similar with `Socks5` in it ?